examples/button-title.html

```diff
@@ -4,6 +4,7 @@
 shortdesc: This example shows how to customize the buttons tooltips with Bootstrap.
 docs: >
   This example shows how to customize the buttons tooltips with <a href="https://getbootstrap.com/docs/4.5/components/tooltips/">Bootstrap</a>.
+  To keep working on fullscreen mode the container option of the tooltip() is required with the value of the target option of the Map().
```

Comment:
I would say “For the tooltips to work in fullscreen mode, set the `container` property to a selector that matches the map `target`.”

Reply:
@tschaub thank you .. I will modify but now i have to find the "clean" way with github to modify because now my branch is behind... i still dont understand the clean process to do this ...

Reply:
If the remote repository name is called `openlayers` you can run these commands, if you named the remote differently you  need to replace `openlayers`  with what you called it:
1. if you have uncommitted changes you need to stash  them: `git stash`
2. Checkout this branch if you are not already there: `git checkout tooltipbsfullscreen`
3. Load the latest changes: `git fetch openlayers`
3. Rebase your branch on the latest changes `git rebase openlayers/main`
4. Unstash your changes if you had any in 1.: `git stash pop`

Reply:
@MoonE  thanks
if i use GitHub Desktop i have a command to rebase branch so the dialog tell me :  
"This will update tooltipbsfullscreen by applying its 22 commits on top of origin/main"
is it right ?
and must i make the modification proposed by @tschaub before ? or after the rebase ?

Reply:
I don't use GitHub Desktop, but the command sounds right. But you need to update your origin/main first!
Your origin/main is this: https://github.com/jipexu/openlayers/commits/main
If you haven't already, you should update your origin/main with the latest changes of openlayers/main first before running the rebase command and remove changes you made there or create a branch for them, otherwise these changes will be included in this branch.

You can do additional changes before or after. It shouldn't matter.

Reply:
i am affraid to have done something wrong now ....